README.adoc

```diff
@@ -174,7 +169,7 @@ Expect the following response headers and values if the simple CORS request is s
 
 Since you started Open Liberty in development mode at the start of the guide, press the `enter/return` key to run the tests.
 
-The [hotspot=testSimpleCorsRequest file=1]`testSimpleCorsRequest` test passes and prints the response headers with their values from the endpoint. The
+If the [hotspot=testSimpleCorsRequest file=1]`testSimpleCorsRequest` test passes, it will print the response headers with their values from the endpoint. The
```

Comment:
Update

>If the `testSimpleCorsRequest` test passes, it will print the response headers with their values from the endpoint.

To

>If the `testSimpleCorsRequest` test passes, the response headers with their values from the endpoint are printed.

Reply:
Addressed.